pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/thread/ThreadRuntime.java

```diff
@@ -119,24 +118,24 @@ private static ClassLoader getFunctionClassLoader(InstanceConfig instanceConfig,
                                                       String narExtractionDirectory,
                                                       FunctionCacheManager fnCache,
                                                       Optional<ConnectorsManager> connectorsManager,
-                                                      Optional<FunctionsManager> functionsManager,
-                                                      Function.FunctionDetails.ComponentType componentType)
```

Comment:
@nicoloboschi , On apache 2_10 component type is initialised in the function and not using the argument.
Can you have a look and let me know if its okay to remove the componentType parameter from the function.

The 2 places where this function getFunctionClassLoader is called are -
-  https://github.com/datastax/pulsar/pull/228/files#diff-6ada125186394b5be32c914e7dca3d43635a33bb81ab4ff81bb3153616e86953L208
- https://github.com/datastax/pulsar/pull/228/files#diff-6ada125186394b5be32c914e7dca3d43635a33bb81ab4ff81bb3153616e86953R200

Reply:
as long as it comes from the same logic, it's okay to drop the function arg